Assets/MixedRealityToolkit/Providers/Hands/HandRay/IHandRay.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.MixedReality.Toolkit.Utilities;
```

Comment:
Needs license header.

Reply:
I just added the license header (it looks like @provencher has the setting that allows for contributor commits for open PR branches on his fork).

I'm dismissing the stale review and getting this change in now.